app.py

```diff
@@ -50,7 +50,7 @@ def create_transaction():
       'private_key': request.args.get('private_key', type=str)
     }
     transaction = blockchain.create_transaction(transaction_data)
-    return jsonify({'transaction_hash': transaction.get_hash()}), 200
+    return jsonify({'data': transaction.get_hash()}), 200
```

Comment:
it would return like `{'data': '833d4115601ba340f329a632f956aa685537b6cb05d4629bab1828382f07e7df'}` which wouldn't be clear

Reply:
Sure